app/containers/App.module.scss

```diff
@@ -15,3 +15,20 @@
   display: flex;
   flex-direction: row;
 }
+
+.externalLinks {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-right: 0.5rem;
+
+  .link {
+    color: var.$c-white;
+    padding: 0.4rem 1rem;
+
+    &:hover {
+      color: var.$c-white;
+      background: rgba(138, 155, 168, 0.15);
```

Comment:
@oluwakemi-dada  When we add some new color to the pallete, it'd be better to add some variable to `_variables.scss`, e.g. `$c-blue-100`. Then the color shades should be specified without alpha parameter (e.g. take a look at $c-green-80, $c-green-60 ... etc.). The opacity in the elements background can cause many problems, when multiple elements overlap each other.
But, in case you need to use rgba value somewhere, it's better to do it using sass rgba() function, e.g:
`background-color: rgba($color: $c-some-color, $alpha: 0.5);`

Also, is there specific reason why we add this new color to the color pallete? Maybe we could use some shade of $c-green-.. color? I'd stick to the current color pallete, and extend it only if it's justified

Reply:
I can't remember why i didn't make it a variable. I will make sure to follow this next time.
I didn't use existing color because i wanted the shade for the hover to be the same as the hover shade for the globe at the top right corner of the page. But couldn't find the color among the variables.

This style has been removed though. I updated the PR.
Thanks for the detailed review :) @jchojna